tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -23,5 +23,5 @@ tests:
     extra_args: OVERLAY_CONFIG="overlay-stm32-spi-dma.conf"
     harness_config:
       fixture: spi_loopback
-    platform_allow: nucleo_g474re nucleo_f207zg nucleo_f429zi nucleo_wb55rg
-        nucleo_f746zg nucleo_g474re nucleo_wb55rg
+    platform_allow: nucleo_g474re nucleo_f207zg nucleo_f429zi
+        nucleo_f746zg nucleo_wb55rg nucleo_h743zi
```

Comment:
Please move `nucleo_wb55rg` and `nucleo_g474re` duplicated erase to first commit, or fix current commit message